lib/lighthouse_bgs/services/benefit.rb

```diff
@@ -49,5 +55,175 @@ def cancel_end_product(file_number:, end_product_code:, modifier:, reason: '1')
 
       response.body || []
     end
+
+    # either date_of_claim OR suspense_date must be populated mm/dd/yyyy
+    def insert_benefit_claim(
```

Comment:
Can we pass these in as a hash instead of this many method args? maybe we can add a validation method for required keys.

Reply:
That makes sense. We can work on that.

Reply:
@charleystran Good afternoon.

I wanted to get a 'gut check' from you on how we implement missing hash parameters. Below are links to an implementation. I wanted to make sure we were on the right track before going through all of the calls in this PR.

If everything is to your liking, we'll move through the calls and get this buttoned up.

## Method in base to validate hash
https://github.com/department-of-veterans-affairs/lighthouse-bgs/pull/19/files#diff-d07e3597702fe5415685adab710f692b

## Example of implmentation
https://github.com/department-of-veterans-affairs/lighthouse-bgs/pull/19/files#diff-c982cf41898881885f99f9d3d5fd21ccR19

Reply:
Yea, I think we can do that for now. I am a little torn because I almost feel like that logic belongs client side vs gem side. Since we dont control what the BGS validations are, they could change and we would want the new validations properly sent back to the client. However for now, I think we can start this way and adjust.

Reply:
@charleystran Hey there, sorry for dragging this PR out. We've updated all the calls with the hash method signature. This should be ready to go. What do you think?